src/Illuminate/Http/Client/Factory.php

```diff
@@ -162,6 +169,21 @@ public function assertSent($callback)
         );
     }
 
+    /**
+     * Assert that every created response sequence is empty.
+     *
+     * @return void
+     */
+    public static function assertSequencesAreEmpty()
```

Comment:
Is there a reason you made this and the property static?

Reply:
The property has to be static because `sequence()` is static.

There is no specific reason I made the assertion static.

Reply:
Ah, I see why.